bokehjs/src/lib/models/canvas/cartesian_frame.ts

```diff
@@ -207,6 +210,41 @@ export class CartesianFrameView extends StyledElementView {
       return {x: x + offset, y: y + offset}
     }
   }
+
+  readonly position = new InlineStyleSheet()
```

Comment:
Could this end up being shared by instances?

Reply:
Ultimately yes, but currently `CartesianFrame` is an oddball, somewhere between a `Renderer` a canvas `UIElement`. I intend to bring positioning under a unified mechanism at some point, so that we don't have custom mechanisms in different parts of UI class hierarchy.

Reply:
Then, I would comment that this is a risk until your intention becomes reality.